src/ganon/acmd/specials/special_high.rs

```diff
@@ -1,7 +1,7 @@
 //! This function just deals with the ground special attack's startup. How the rest of
 //! the move is handled belongs to `crate::ganon::omni_teleport`.
```

Comment:
Is this correct? This docstring is the same as [`special_neutral.rs`](https://github.com/ZeDespo/Ultimate-Ganondorf/blob/420ed848507d14e5d9bad95db9ece0d3bda88107/src/ganon/acmd/specials/special_neutral.rs#L1)

Also, where is `omni_teleport`? Is that something that is currently being worked on, but doesn't "exist" yet?

Just to make sure I understand things correctly, when exactly does this special high acmd run? Because according to the [teleport status script](https://github.com/ZeDespo/Ultimate-Ganondorf/blob/420ed848507d14e5d9bad95db9ece0d3bda88107/src/ganon/status/teleport.rs), you're making special hi change into the `FIGHTER_GANON_STATUS_KIND_PRE_TELEPORT` status, and no where in that you are changing the motion to `special_hi`.

Reply:
- The docstring should NOT be the same. Originally, Special Neutral was going to act as the teleport but it didn't make sense given how it operates like a recovery move. The neutral special spot is still up for grabs. 
- `omni_teleport` was the old name for this functionality until I decided to rename it to `teleport`.
- Special High ACMD runs during the teleport. I can't explain why, but the ACMD for special_high is still applied. Maybe it's because the status kind is not tied to an entry in `motion_list.yml`, smash resolves the hitboxes for special_hi anyways? It's a guess on my part.

Reply:
It might be because you're doing `StatusModule::change_status_request_from_script`, it's probably going through with the normal special hi, and then changing to your teleport status at little later on.

Reply:
That's a much simpler and better explanation.